libs/ledger-live-common/src/families/polkadot/walletApiAdapter.ts

```diff
@@ -0,0 +1,27 @@
+import { PolkadotTransaction as WalletAPITransaction } from "@ledgerhq/wallet-api-core";
+import { Transaction } from "./types";
+
+const CAN_EDIT_FEES = false;
+
+const convertToLiveTransaction = (
+  tx: WalletAPITransaction
+): Partial<Transaction> => ({
+  ...tx,
+  era: tx.era ? `${tx.era}` : undefined,
+});
+
+const getWalletAPITransactionSignFlowInfos = (
```

Comment:
Some functions seems to always have the same prototype, would it be a good idea to define a type and import it in all those adapter modules ?

Reply:
Indeed the `getWalletAPITransactionSignFlowInfos` function has the same signature for all coins needing an adapter as of today. The type can indeed be extracted 👍

Reply:
cf. 6d3fea8d1dce8e07eff8b571cc9b3a017a4090b5 ✅